src/ImageSharp/Formats/Tiff/TiffMetadata.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.Formats.Experimental.Tiff.Constants;
+
+namespace SixLabors.ImageSharp.Formats.Experimental.Tiff
+{
+    /// <summary>
+    /// Provides Tiff specific metadata information for the image.
+    /// </summary>
+    public class TiffMetadata : IDeepCloneable
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TiffMetadata"/> class.
+        /// </summary>
+        public TiffMetadata()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TiffMetadata"/> class.
+        /// </summary>
+        /// <param name="other">The metadata to create an instance from.</param>
+        private TiffMetadata(TiffMetadata other)
+        {
+            this.ByteOrder = other.ByteOrder;
+            this.XmpProfile = other.XmpProfile;
+            this.BitsPerPixel = other.BitsPerPixel;
+        }
+
+        /// <summary>
+        /// Gets or sets the byte order.
+        /// </summary>
+        public ByteOrder ByteOrder { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of bits per pixel.
+        /// </summary>
+        public TiffBitsPerPixel BitsPerPixel { get; set; } = TiffBitsPerPixel.Pixel24;
+
+        /// <summary>
+        /// Gets or sets the compression used to create the TIFF file.
+        /// </summary>
+        public TiffCompression Compression { get; set; } = TiffCompression.None;
+
+        /// <summary>
+        /// Gets or sets the XMP profile.
+        /// </summary>
+        public byte[] XmpProfile { get; set; }
```

Comment:
Shouldn't we have a dedicated type for this?

Reply:
@JimBobSquarePants ImageSharp don't support XMP profile yet, so here the data is in raw format

Reply:
This should be internal then. We cannot introduce a change we will likely break

Reply:
i will make that internal tomorrow

Reply:
@JimBobSquarePants @brianpopow didn't notice this message, yes, it's done!